src/autoyast-rnc/samba-server.rnc

```diff
@@ -481,6 +484,6 @@ samba_options = element options {
   MAP,
   element option { MAP, samba_option_key, samba_option_val? }*
 }
-share = element share { (STRING | y2_share)* }
+share = element share { (text | y2_share)* }
```

Comment:
this channge looks a bit strage. Does it mean that type string is not allowed here?